app/views/projects/samples/index.html.erb

```diff
@@ -58,6 +58,16 @@
         class: "button button--size-default button--state-primary",
         "aria-label": t(".actions.button_add_aria_label") %>
       <% end %>
+      <% if allowed_to?(:destroy_sample?, @project) %>
+       <%= link_to t(".delete_samples_button"),
+        new_destroy_multiple_namespace_project_samples_path,
+        data: {
+          turbo_stream: true,
+          controller: "action-link",
+          action_link_required_value: 1,
+        },
+        class: "button button--size-default button--state-destructive action-link" %>
```

Comment:
**nitpick:** Can you use the tailwind classes directly, I am moving to get rid of that `application.tailwind.css` file

Reply:
Changed in 7d391821c2145a485846f5eaa33da0f018e9dcae